global_tf:
  process_freq: 50.0
  is_real: true
  extrinsic_imu_to_camera_x_m: 0.0
  extrinsic_imu_to_camera_y_m: 0.0
  extrinsic_imu_to_camera_z_m: 0.0
  extrinsic_imu_to_camera_r_deg: 0.0
  extrinsic_imu_to_camera_p_deg: 180.0
  extrinsic_imu_to_camera_y_deg: 90.0